.github/workflows/build.yml

```diff
@@ -9,7 +9,8 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [8.x, 10.x, 12.x, 14.x, 16.x]
+        node-version: [8.x, 10.x, 12.x, 14.x, 16.x, 18.x]
+      fail-fast: false
```

Comment:
The fail fast bit isn't necessary but it gives a clearer picture of which node versions are failing.